include/yaramod/parser/parser_driver.h

```diff
@@ -180,7 +180,9 @@ class ParserDriver
 	std::string _indent; ///< Variable storing current indentation
 	std::string _comment; ///< For incremental construction of parsed comments
 	std::string _regexpClass; ///< Currently processed regular expression class.
+	size_t _regexpClassDepth = 0; ///< The rectangular brackets depth of nesting
```

Comment:
`std::size_t` instead of `size_t` to follow the conventions used in other parts of yaramod code.